repose-aggregator/components/filters/destination-router-filter/src/test/scala/org/openrepose/filters/destinationrouter/DestinationRouterFilterTest.scala

```diff
@@ -38,13 +38,19 @@ import org.springframework.mock.web.{MockFilterChain, MockFilterConfig, MockHttp
 @RunWith(classOf[JUnitRunner])
 class DestinationRouterFilterTest extends FunSpec with Matchers with BeforeAndAfterEach with MockitoSugar {
 
-  val configurationService = mock[ConfigurationService]
-  val meterByCategorySum = mock[MeterByCategorySum]
-  val metricsService = mock[MetricsService]
+  private val configurationService = mock[ConfigurationService]
+  private val metricsService = mock[MetricsService]
+  private val metricRegistry = mock[MetricRegistry]
+  private val meter = mock[Meter]
 
-  override def beforeEach() = {
+  override def beforeEach(): Unit = {
     reset(configurationService)
     reset(metricsService)
+    reset(metricRegistry)
+    reset(meter)
```

Comment:
We should probably discuss this at a Dev Meeting and then clean up this behavior across the board.